pkg/kubectl/cmd/log.go

```diff
@@ -82,8 +84,13 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"log"},
 	}
 	cmd.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
+	cmd.Flags().Bool("timestamps", false, "Include timestamps on each line in the log output")
 	cmd.Flags().Bool("interactive", true, "If true, prompt the user for input when required. Default true.")
 	cmd.Flags().BoolP("previous", "p", false, "If true, print the logs for the previous instance of the container in a pod if it exists.")
+	cmd.Flags().Int("limit-bytes", 0, "Maximum bytes of logs to return. Defaults to no limit.")
```

Comment:
Add a couple of examples that exercise the new flags and while you are here, please fix the existing examples to use `-c` for specifying the container. Also changing the pod names to something more readable than 123456-7890 would be nice.